code/game/objects/items/weapons/material/knives.dm

```diff
@@ -95,13 +95,21 @@
 
 /obj/item/weapon/material/knife/machete
 	name = "machete"
-	desc = "A sharp machete often found in survival kits."
+	desc = "A long, sturdy blade with a rugged handle. Leading the way to cursed treasures since before space travel." //VOREStation Edit
 	icon_state = "machete"
 	force_divisor = 0.3 // 18 when hardness 60 (steel)
 	attack_verb = list("slashed", "chopped", "gouged", "ripped", "cut")
 	can_cleave = TRUE //Now hatchets inherit from the machete, and thus knives. Tables turned.
 	slot_flags = SLOT_BELT
-	default_material = "plasteel"
+	default_material = "plasteel" //VOREStation Edit

```

Comment:
Or this, but why is this here? Is this a Vorestation edit or original code? Why add the Vorestation Edit tag where this was original Polaris code?